README.md

```diff
@@ -62,6 +62,7 @@ The table below identifies the services this tool supports and some example serv
 | [NextcloudTalk](https://github.com/caronc/apprise/wiki/Notify_nextcloudtalk) | nctalk:// or nctalks:// | (TCP) 80 or 443 | nctalk://user:pass@host/RoomId<br/>nctalks://user:pass@host/RoomId1/RoomId2/RoomIdN
 | [Notica](https://github.com/caronc/apprise/wiki/Notify_notica) | notica://  | (TCP) 443   | notica://Token/
 | [Notifico](https://github.com/caronc/apprise/wiki/Notify_notifico) | notifico://  | (TCP) 443   | notifico://ProjectID/MessageHook/
+| [Ntfy](https://github.com/caronc/apprise/wiki/Notify_ntfy) | ntfy://  | (TCP) 80 or 443   | ntfy://topic/<br/>ntfys://topic/
```

Comment:
`[ntfy]...`

Reply:
It just feels so wrong compared to all the others :sob:

Reply:
It's VMware, jQuery, and the Web (as opposed to "web"). Those who invent it get to name it. ;-)

To me it looks wrong spelling it Ntfy. Android makes me do it but everywhere else I can spell it the correct way.